### PR TITLE
enhance: add base class 'OVSvsctlList'

### PR DESCRIPTION
**Pull request by shlao**
enhance: add a basic class 'OVSvsctlList'  for parser 'OVSvsctlListBridge'

Signed-off-by: shlao <shlao@redhat.com>

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [x] Is this PR an enhancement?

### Complete Description of Additions/Changes:

There is an OVN rule that needs a parser to parser the output from the command `/usr/bin/ovs-vsctl list Open_vSwitch`, I find that we could write a basic class `OVSvsctlList`, and used it in the parsers: `OVSvsctlListBridge`, `OVSvsctlListOpenvSwitch`. please see: CEECBA-6561 card.

### CODE REVIEW

**Comment by shlao** [comment]
@xiangce  Please, have a look , Thanks!

File: insights/parsers/ovs_vsctl_list_bridge.py [hunk 1]
```diff
@@ -4,16 +4,28 @@
...
+
+.. warning::
+    This parser `OVSvsctlListBridge` is deprecated, please use
+    :py:class:`insights.ovs_vsctl.OVSvsctlListBridge` instead.
 """
 
-from insights import CommandParser, get_active_lines, parser
-from insights.parsers import SkipException, optlist_to_dict
+from insights import parser
+from insights.parsers.ovs_vsctl import OVSvsctlList
```
Comment (xiangce) [inline insights/parsers/ovs_vsctl_list_bridge.py:18]: You may want to use the `util.deprecated` function to mark this parser will be deprecated in the future version , see  https://github.com/RedHatInsights/insights-core/blob/master/insights/parsers/journal_since_boot.py#L55

Comment (shlao) [reply insights/parsers/ovs_vsctl_list_bridge.py:18]: @xiangce Thanks, Updated it, please have a look.

**Review by xiangce — APPROVED** [review]

**Review by xiangce — APPROVED** [review]
Looks good to me now.

**Pull request merged**